packages/shared/scripts/postinstall.js

```diff
@@ -1,26 +1,26 @@
 const fs = require("fs");
 const path = require("path");
 const currentDir = path.dirname(__dirname);
-const stylesPath = path.resolve("../../../sfui.scss");
+const stylesPath = path.resolve(__dirname, "../../../sfui.scss");
+const variablePath = path.resolve(__dirname, "../styles/_variables.scss");
 const isDependency = currentDir.includes("node_modules");
-
 const stylesContent = `/* Here you can override global SCSS variables */
   
 // global override excluding scoped components
 // $button-desktop-padding: 5px !default;
 // global override including scoped componentss
 // $button-desktop-padding: 50px;`;
 
-const playgroundContent = `<template>
-  <div id="playground"></div>
-</template>
-  
-<script>
-// Use this component to play with other components
-export default {};
-</script>`;
 
 if (isDependency && !fs.existsSync(stylesPath)) {
+  fs.readFile(variablePath, "utf8", function(err, data) {
```

Comment:
We shouldn't modify node_modules directory. Default installation should include proper path.

Reply:
We should ship working package to every user, not package that works after running a script. Path overriding must happen before publishing and reverse itself after.

Reply:
Thanks @filrak